KebPOS/Services/ProductService.cs

```diff
@@ -1,180 +1,211 @@
-                        The pot is broken while eating. 
-                        We all know how special the taste of “matke ka pani” is. 
-                        Similarly, the taste of matka Kebab is very special.
-                    """,
-                    Price = 3.69m
-                },
-                new Product()
-                {
-                    Id = 9,
-                    Name = "Dill Salmon Kebab",
-                    Description = """
-                        Dill salmon Kebab is very special Kebab for all seafood lovers and especially for fish lovers. 
-                        The dish is very yummy.
-                    """,
-                    Price = 3.99m
-                },
-                new Product()
-                {
-                    Id = 10,
-                    Name = "Lamb Kebab",
-                    Description = """
-                        Lamb Kebabs are very easy to make. 
-                        What all you need to do is marinate the mince meat with all the spices. 
-                        You can add egg also just to enhance the taste of Kebab.
-                    """,
-                    Price = 3.79m
-                },
-                new Product()
-                {
-                    Id = 11,
-                    Name = "Cappuccino",
-                    Description = "A freshly pulled shot of espresso layered with steamed whole milk and thick rich foam to offer a luxurious velvety texture and complex aroma.",
-                    Price = 1.49m
-                },
-                new Product()
-                {
-                    Id = 12,
-                    Name = "Red Bull",
-                    Description = "Red Bull is a utility drink to be taken against mental or physical weariness or exhaustion.",
-                    Price = 1.87m
-                },
-                new Product()
-                {
-                    Id = 13,
-                    Name = "Coca Cola",
-                    Description = "Coca-Cola is a carbonated, sweetened soft drink and is the world's best-selling drink. A popular nickname for Coca-Cola is Coke.",
-                    Price = 0.93m
-                },
-                new Product()
-                {
-                    Id = 14,
-                    Name = "Sprite",
-                    Description = "Crisp, refreshing and clean-tasting, Sprite is a lemon and lime-flavoured soft drink.",
-                    Price = 1.99m
-                },
-                new Product()
-                {
-                    Id = 15,
-                    Name = "Bonaqua Sparkling",
-                    Description = "BonAqua is a high-quality drinking water.",
-                    Price = 1.24m
-                }
-            };
-            return products;
+            bool duplicate = false;
+            var product = new Product();
+            product = new Product();
+            product.Name = AnsiConsole.Ask<string>("Product's name:");
```

Comment:
At the moment I can add negative numbers to the price and an enormous amount of characters for product's name and description. It would be good to limit the product's name to 20 characters and the description to 200 characters. You'll need to create new methods in the validation class for this.

- Check if the string isn't null or empty and number of characters is within the limit
- Check if string can be parsed to decimal and is >=0

Reply:
Question:  Spectre console won't let you enter an empty string so I didn't write a check to see if the string is null or empty.  Is it best practice to still check.  Just wondering the why.

Reply:
> Question: Spectre console won't let you enter an empty string so I didn't write a check to see if the string is null or empty. Is it best practice to still check. Just wondering the why.

Good point. Up to you. I guess checking would allow you to return an error message to the user for better UX, but I'm not too fussed